app/controllers/discourse_reactions/custom_reactions_controller.rb

```diff
@@ -136,19 +135,24 @@ def reactions_received
   end
 
   def post_reactions_users
-    id = params.require(:id).to_i
+    params.require(:id).to_i
```

Comment:
what is the point of `to_i` here?

Reply:
No idea TBH I just removed the assignment to get rid of the rubocop warning :) will remove to_i as well